### DISCUSSION TITLE
Maximum measurable habit

### DISCUSSION THREAD

**Discussion by SamJ221** [iSoron/uhabits#1389] [Feature Requests]
Good day, I am absolutely loving the app, thanks for all the work you put into it.

 I was hoping to see a maximum frequency added. Where the goal is complete until you go over the set number then it counts as a fail. IE. I don't want to drink more that 3 coffees a day, 0,1,2,3 are acceptable and mark the goal complete but 4+ will mark the goal as not accomplished

Thank you so much.

**Comment by hunterwilhelm** [comment]
Along those lines I want to be able to track bad habits with the measurements. So if I go over zero that's bad. Or perhaps could we allow negative numbers as an option in the settings?

**Comment by SP-cell62** [comment]
Same for measurable stuff. So or example I might want a tick if I ate LESS than a certain amount of calories, like I get tick for walking more than a target amount of steps.

**Comment by herrbrixter** [comment]
Related to #696
